src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Account/NotificationSettings.cshtml

```diff
@@ -28,7 +28,7 @@
 
                         <tr>
                             <td class="notification-row">
-                                @setting.UserRef
+                                @Model.Email
```

Comment:
Could we instead use the UserRef data when it isn't a GUID and fall back to the email when it is? The user account I used had an email in the format `oattestuser+10008426@gmail.com` which doesn't look great but may just be test data.